operations/mimir-mixin/docs/playbooks.md

```diff
@@ -1130,6 +1130,133 @@ The limit protects the system’s stability from potential abuse or mistakes, an
+The limit is a per-instance limit and it's applied on all samples received, across all tenants, in each ingester.
+
+How to **fix**:
+- Scale up ingesters.
+- Increase the limit by using the `-ingester.instance-limits.max-ingestion-rate` option (or `max_ingestion_rate` in the runtime config).
+
+### err-mimir-ingester-max-tenants
+
+This critical error occurs when the ingester receives a write request for a new tenant (a tenant for which no series have been stored yet) but the ingester cannot accept it because the maximum number of allowed tenants per ingester has been reached.
+
+How to **fix**:
+- In case of emergency, increase the limit by using the `-ingester.instance-limits.max-tenants` option (or `max_tenants` in the runtime config).
+- Consider configuring ingesters shuffle sharding to reduce the number of tenants per ingester.
+
+### err-mimir-ingester-max-series
+
+This critical error occurs when an ingester rejects a write request because it reached the maximum number of in-memory series.
+
+How it **works**:
+
+- The ingester keeps most recent series data in-memory.
+- The ingester has a per-instance limit on the number of in-memory series, used to protect the ingester from overloading in case of high traffic.
+- When the number of in-memory series, new series are rejected, while samples can still be appended to existing ones.
+- You can configure the limit by setting the `-ingester.instance-limits.max-series` option (or `max_series` in the runtime config).
+
+How to **fix**:
+- See [`MimirIngesterReachingSeriesLimit`](#MimirIngesterReachingSeriesLimit) runbook.
+
+### err-mimir-ingester-max-inflight-push-requests
+
+This error occurs when an ingester rejects a write request because the max inflight requests limit has been reached.
+
+How it **works**:
+- The ingester has per-instance limit on the number of inflight write (push) requests.
+- The limit applies on all inflight write requests, across all tenants, and is used to protect the ingester from overloading in case of high traffic.
+- You can configure the limit by setting the `-ingester.instance-limits.max-inflight-push-requests` option (or `max_inflight_push_requests` in the runtime config).
+
+How to **fix**:
+- In case of emergency, increase the limit by setting the `-ingester.instance-limits.max-inflight-push-requests` option (or `max_inflight_push_requests` in the runtime config).
+- Check the write requests latency through the `Mimir / Writes` dashboard and eventually investigate the root cause of high latency (the higher the latency, the higher the number of inflight write requests).
+- Consider scaling out the ingesters.
+
+### err-mimir-max-series-per-user
+
+This error occurs when the number of in-memory series for a given tenant exceeds the configured limit.
+
+The limit is used to protect ingesters from overloading in case a tenant writes an high number of series, as well as to protect the whole system’s stability from potential abuse or mistakes.
```

Comment:
```suggestion
The limit is used to protect ingesters from overloading in case a tenant writes a high number of series, as well as to protect the whole system’s stability from potential abuse or mistakes.
```